test/recipes/95-test_external_krb5_data/krb5.sh

```diff
@@ -13,7 +13,7 @@ CFLAGS="-I`pwd`/$BLDTOP/include -I`pwd`/$SRCTOP/include"
 
 cd $SRCTOP/krb5/src
 autoreconf
-./configure --with-ldap --with-prng-alg=os --enable-pkinit \
+./configure --with-ldap --with-prng-alg=os --disable-pkinit \
```

Comment:
Why is it necessary to disable PKINIT?
The PKINIT code is one place in krb5 that has a hard dependency on openssl.

Do I need to spin up my test setup and investigate?

Reply:
Looks like the first batch of errors is due to const issues:

```
main.c: In function 'add_pubkey_info':
main.c:430:9: warning: 'EVP_PKEY_get0_RSA' is deprecated: Since OpenSSL 3.0 [-Wd         rsa = EVP_PKEY_get0_RSA(key);st_external_krb5/../../include/openssl/evp         ^~~note: declared here
In file included from main.c:38:0:0_RSA(const EVP_PKEY *pkey);
/home/bkaduk/git/opensslb/test-runs/test_external_krb5/../../include/openssl/evp const struct rsa_st *EVP_PKEY_get0_RSA(const EVP_PKEY *pkey);om pointer target                       ^~~~~~~~~~~~~~~~~
```

7bc0fdd3fd4535e06c35b92d71afab9a6de94cc5 is at least clear that the OTC decided to make this breaking change, and the krb5 tests confirm that it is a breaking change.